modules/mono/editor/GodotTools/GodotTools/Build/MSBuildPanel.cs

```diff
@@ -72,12 +72,12 @@ private void BuildMenuOptionPressed(long id)
             }
         }
 
-        public void BuildProject()
+        public async void BuildProject()
```

Comment:
We should probably return `Task` here and discard it in `BuildMenuOptionPressed`.

```suggestion
        public async Task BuildProject()
```